src/app.ts

```
@@ -91,11 +92,10 @@ import {
 import { FeedUpdateService } from './feed/feed-update.service';
 import { FeedService } from './feed/feed.service';
 import { FlagService } from './flag/flag.service';
+import { getSiteDataHandler, inboxHandler } from './handlers';
 import {
     createDerepostActionHandler,
```

Biome (1.9.4)

[error] Shouldn't redeclare 'createDerepostActionHandler'. Consider to delete it or rename it.

'createDerepostActionHandler' is defined here:


(lint/suspicious/noRedeclare)